alpine/entrypoint.sh

```diff
@@ -12,4 +12,4 @@ if traefik "$1" --help 2>&1 >/dev/null | grep "help requested" > /dev/null 2>&1;
     set -- traefik "$@"
 fi
 
-exec "$@"
\ No newline at end of file
+exec $(eval printf '%s\ ' "$@")
```

Comment:
One last shellcheck (sorry :) ): 
```
$ shellcheck ./entrypoint.sh

In /Users/dadou/test.sh line 5:
exec $(eval printf '%s\ ' "$@")
     ^-- SC2046: Quote this to prevent word splitting.
```

=> Should be `exec "$(eval printf '%s\ ' "$@")"`

Reply:
It Does not work, 
I wrote in the commit info. if i will correct the last shellcheck error i will break the entire script

Reply:
`exec "$(eval printf '%s\ ' "$@")"`

same output when using both echo and printf

```bash
+ [ -metrics.datadog.address="${PROXY_SERVICE_HOST}:8125" != --metrics.datadog.address="${PROXY_SERVICE_HOST}:8125" ]
+ set -- traefik --metrics.datadog.address="${PROXY_SERVICE_HOST}:8125"
+ traefik traefik --help
+ grep help requested
+ eval printf %s\  traefik --metrics.datadog.address="${PROXY_SERVICE_HOST}:8125"
+ printf %s  traefik --metrics.datadog.address=10.107.59.10:8125
+ exec traefik --metrics.datadog.address=10.107.59.10:8125 
/entrypoint.sh: exec: line 1: traefik --metrics.datadog.address=10.107.59.10:8125 : not found
```